Source/Core/Core/CoreTiming.h

```diff
@@ -37,7 +37,7 @@ namespace CoreTiming
 struct Globals
 {
   s64 global_timer = 0;
-  int slice_length = 0;
+  s64 slice_length = 0;
```

Comment:
This change is also strange. `slice_length` is capped to `MAX_SLICE_LENGTH` in `Advance()` (which is 20000, so easily fits into an `int`), so why does it need to be an `s64`?